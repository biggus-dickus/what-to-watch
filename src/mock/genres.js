export const genres = [`Comedies`, `Crime`, `Documentary`, `Dramas`, `Horror`, `Kids & Family`, `Romance`, `Sci-Fi`, `Thrillers`, `Good ol' Porn`];
