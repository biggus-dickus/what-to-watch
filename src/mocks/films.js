export const films = [
  {
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    picUrl: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Bohemian Rhapsody`,
    picUrl: `img/bohemian-rhapsody.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Macbeth`,
    picUrl: `img/macbeth.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Aviator`,
    picUrl: `img/aviator.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `We need to talk about Kevin`,
    picUrl: `img/we-need-to-talk-about-kevin.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `What We Do in the Shadows`,
    picUrl: `img/what-we-do-in-the-shadows.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Revenant`,
    picUrl: `img/revenant.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Johny English`,
    picUrl: `img/johnny-english.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Shutter Island`,
    picUrl: `img/shutter-island.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Pulp Fiction`,
    picUrl: `img/pulp-fiction.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `No Country for Old Men`,
    picUrl: `img/no-country-for-old-men.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Snatch`,
    picUrl: `img/snatch.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Moonrise Kingdom`,
    picUrl: `img/moonrise-kingdom.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Seven Years in Tibet`,
    picUrl: `img/seven-years-in-tibet.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Midnight Special`,
    picUrl: `img/midnight-special.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `War of the Worlds`,
    picUrl: `img/war-of-the-worlds.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Dardjeeling Limited`,
    picUrl: `img/dardjeeling-limited.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Orlando`,
    picUrl: `img/orlando.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
  {
    name: `Mindhunter`,
    picUrl: `img/mindhunter.jpg`,
    pageUrl: `https://thepiratebay.org`
  },
];
