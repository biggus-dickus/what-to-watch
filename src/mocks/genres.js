export const Genre = {
  ALL: `All genres`,
  COMEDIES: `Comedies`,
  CRIME: `Crime`,
  DOCUMENTARY: `Documentary`,
  DRAMAS: `Dramas`,
  HORROR: `Horror`,
  FAMILY: `Kids & Family`,
  ROMANCE: `Romance`,
  SCI_FI: `Sci-Fi`,
  THRILLERS: `Thrillers`
};
