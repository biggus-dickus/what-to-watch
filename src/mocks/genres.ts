import {Genre as GenreType} from '../types'; // eslint-disable-line

export const Genre = {
  ALL: `All genres`,
  COMEDIES: `Comedies`,
  CRIME: `Crime`,
  DOCUMENTARY: `Documentary`,
  DRAMAS: `Dramas`,
  HORROR: `Horror`,
  FAMILY: `Kids & Family`,
  ROMANCE: `Romance`,
  SCI_FI: `Sci-Fi`,
  THRILLERS: `Thrillers`
};

export const mockGenres = Object.values(Genre);

export const mockGenreEntity: GenreType = {
  currentGenre: Genre.CRIME,
  genres: [...mockGenres],
  onGenreChange: jest.fn()
};
