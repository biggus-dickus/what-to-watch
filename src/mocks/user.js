const mockUser = {
  id: 1,
  email: `Oliver.conner@gmail.com`,
  name: `Oliver.conner`,
  avatar_url: `img/1.png` // eslint-disable-line
};

export default mockUser;
