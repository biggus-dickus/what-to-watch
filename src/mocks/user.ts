import {Location, User} from '../types'; // eslint-disable-line

export const mockLocation: Location = {
  hash: `#p1t00h`,
  key: `0mfg666`,
  pathname: `/`,
  search: ``
};

const mockUser: User = {
  avatar_url: '/wtw/static/avatar/10.jpg', // eslint-disable-line
  id: 777,
  email: `test@test.com`,
  name: `Nastasha Romanenko`,
};

export default mockUser;
