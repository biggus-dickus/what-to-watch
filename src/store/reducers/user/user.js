import {ActionType} from '../../action-types';

const initialState = {
  isAuthorizationRequired: false,
  userData: null
};

export const userReducer = (state = initialState, action = {}) => {
  switch (action.type) {
    case ActionType.AUTH_REQUIRED:
      return {
        ...state,
        isAuthorizationRequired: action.payload
      };

    case ActionType.GET_USER_DATA:
      return {
        ...state,
        userData: action.payload
      };

    default: return state;
  }
};
