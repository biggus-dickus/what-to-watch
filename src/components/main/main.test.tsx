import * as React from 'react';
import {StaticRouter as Router} from 'react-router-dom';
import * as renderer from 'react-test-renderer';

import {mockFilms} from '../../mocks/films';
import {mockGenreEntity} from '../../mocks/genres';
import {mockLocation} from '../../mocks/user';
import mockUser from '../../mocks/user';

import Main from './main';

const props = {
  ...mockGenreEntity,
  isPlayerShown: false,
  location: {...mockLocation},
  movies: mockFilms,
  promo: mockFilms[1],
  onVideoToggle: jest.fn(),
  onWatchListToggle: jest.fn(),
  onViewChange: jest.fn(),
  userData: mockUser
};


it(`renders correctly after relaunch`, () => {
  const tree = renderer.create(<Router><Main {...props} /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
