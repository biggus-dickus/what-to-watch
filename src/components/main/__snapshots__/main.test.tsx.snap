// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly after relaunch 1`] = `
Array [
  <section
    className="movie-card"
  >
    <div
      className="movie-card__bg"
    >
      <img
        alt="The Grand Budapest Hotel"
        height="552"
        src={
          Object {
            "process": [Function],
          }
        }
        width="1300"
      />
    </div>
    <h1
      className="visually-hidden"
    >
      What to Watch
    </h1>
    <header
      className="page-header movie-card__head"
    >
      <div
        className="logo"
      >
        <span
          className="logo__link"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </span>
      </div>
      <div
        className="user-block"
      >
        <a
          className="user-block__avatar"
          href="/mylist"
          onClick={[Function]}
        >
          <img
            alt="Nastasha Romanenko"
            height="63"
            src="https://es31-server.appspot.com/wtw/static/avatar/10.jpg"
            width="63"
          />
        </a>
      </div>
    </header>
    <div
      className="movie-card__wrap"
    >
      <div
        className="movie-card__info"
      >
        <div
          className="movie-card__poster"
        >
          <img
            alt="The Grand Budapest Hotel poster"
            height="327"
            src={
              Object {
                "process": [Function],
              }
            }
            width="218"
          />
        </div>
        <div
          className="movie-card__desc"
        >
          <h2
            className="movie-card__title"
          >
            The Grand Budapest Hotel
          </h2>
          <p
            className="movie-card__meta"
          >
            <span
              className="movie-card__genre"
            >
              Drama
            </span>
            <span
              className="movie-card__year"
            >
              2014
            </span>
          </p>
          <div
            className="movie-card__buttons"
          >
            <button
              className="btn btn--play movie-card__button"
              type="button"
            >
              <svg
                height="19"
                viewBox="0 0 19 19"
                width="19"
              >
                <use
                  xlinkHref="img/sprite/sprite.svg#play-s"
                />
              </svg>
              <span>
                Play
              </span>
            </button>
            <button
              className="btn btn--list movie-card__button"
              type="button"
            >
              <svg
                height="20"
                viewBox="0 0 19 20"
                width="19"
              >
                <use
                  xlinkHref="img/sprite/sprite.svg#add"
                />
              </svg>
              <span>
                My list
              </span>
            </button>
            <a
              className="btn movie-card__button"
              href="/film/666/review"
              onClick={[Function]}
            >
              Add review
            </a>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <div
    className="page-content"
  >
    <section
      className="catalog"
    >
      <h2
        className="catalog__title visually-hidden"
      >
        Catalog
      </h2>
      <ul
        className="catalog__genres-list"
        data-test="at-genres-list"
      >
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#All-genres"
            onClick={[Function]}
          >
            All genres
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#Comedies"
            onClick={[Function]}
          >
            Comedies
          </a>
        </li>
        <li
          className="catalog__genres-item catalog__genres-item--active"
        >
          <a
            className="catalog__genres-link"
            href={null}
            onClick={[Function]}
          >
            Crime
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#Documentary"
            onClick={[Function]}
          >
            Documentary
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#Dramas"
            onClick={[Function]}
          >
            Dramas
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#Horror"
            onClick={[Function]}
          >
            Horror
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#Kids-&-Family"
            onClick={[Function]}
          >
            Kids & Family
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#Romance"
            onClick={[Function]}
          >
            Romance
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#Sci-Fi"
            onClick={[Function]}
          >
            Sci-Fi
          </a>
        </li>
        <li
          className="catalog__genres-item"
        >
          <a
            className="catalog__genres-link"
            href="#Thrillers"
            onClick={[Function]}
          >
            Thrillers
          </a>
        </li>
      </ul>
      <div
        className="catalog__movies-list"
        data-test="at-films-list"
      >
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
                src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Fantastic Beasts: The Crimes of Grindelwald
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/bohemian-rhapsody.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Bohemian Rhapsody
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/macbeth.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4%2FCam-Man.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Macbeth
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/aviator.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/5/5b/Marnie_%281964%29_trailer.webm/Marnie_%281964%29_trailer.webm.360p.vp9.webm"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Aviator
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/we-need-to-talk-about-kevin.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/5/5b/Marnie_%281964%29_trailer.webm/Marnie_%281964%29_trailer.webm.360p.vp9.webm"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                We need to talk about Kevin
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/what-we-do-in-the-shadows.jpg"
                src="https://upload.wikimedia.org/wikipedia/commons/transcoded/2/23/Dr_Strangelove_-_Official_Trailer.webm/Dr_Strangelove_-_Official_Trailer.webm.480p.vp9.webm"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                What We Do in the Shadows
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/revenant.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Harvesting%20Rice.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Revenant
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/johnny-english.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Lake%20Day.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Johny English
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/shutter-island.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Deserted%20Island.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Shutter Island
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/pulp-fiction.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Stunning%20Mountains.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Pulp Fiction
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/no-country-for-old-men.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Gubat%20Beach.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                No Country for Old Men
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/snatch.jpg"
                src="etc/na-zdorovnya.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Snatch
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/moonrise-kingdom.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Caramoan%20Drone.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Moonrise Kingdom
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/seven-years-in-tibet.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Ants%20Walking.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Seven Years in Tibet
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/midnight-special.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Boat%20in%20Caramoan.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Midnight Special
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/war-of-the-worlds.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Blue%20Joy.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                War of the Worlds
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/dardjeeling-limited.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Stop-Sign.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Dardjeeling Limited
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/orlando.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/Steaming-Yellowstone.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Orlando
              </span>
            </h3>
          </a>
        </article>
        <article
          className="small-movie-card catalog__movies-card"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          tabIndex={0}
        >
          <a
            className="small-movie-card__wrapping-link"
            href="/https://thepiratebay.org"
            onClick={[Function]}
          >
            <i
              aria-hidden={true}
              className="small-movie-card__play-btn"
            />
            <div
              className="small-movie-card__image"
            >
              <video
                height={175}
                muted={true}
                poster="img/mindhunter.jpg"
                src="https://storage.googleapis.com/coverr-main/mp4/California-Desert.mp4"
                width={280}
              />
            </div>
            <h3
              className="small-movie-card__title"
            >
              <span
                className="small-movie-card__link"
              >
                Mindhunter
              </span>
            </h3>
          </a>
        </article>
      </div>
    </section>
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <span
          className="logo__link logo__link--light"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </span>
      </div>
      <div
        className="copyright"
      >
        <p>
          © 
          2019
           What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>,
]
`;
